src/ontology/cob-edit.owl

```diff
@@ -1347,6 +1366,58 @@
         <obo:IAO_0000115>A process during which an organism comes into contact with another entity.</obo:IAO_0000115>
         <rdfs:label xml:lang="en">exposure of organism</rdfs:label>
     </owl:Class>
+
+
+
+    <!-- http://purl.obolibrary.org/obo/COB_0001100 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/COB_0001100">
+        <owl:equivalentClass>
+            <owl:Class>
+                <owl:unionOf rdf:parseType="Collection">
+                    <rdf:Description rdf:about="http://purl.obolibrary.org/obo/COB_0001101"/>
+                    <rdf:Description rdf:about="http://purl.obolibrary.org/obo/COB_0001102"/>
+                </owl:unionOf>
+            </owl:Class>
+        </owl:equivalentClass>
+        <rdfs:comment>This class groups the two defined classes &quot;disease&quot; and &quot;disorder&quot;. It supports use cases that conflate the two classes for pragmatic reasons.</rdfs:comment>
```

Comment:
Note, there is no useful way to define this class in a human readable way. But there is a very clear logical definition, which should satisfy any need for further definition.